src/main/java/io/airlift/slice/Slice.java

```diff
@@ -888,7 +888,7 @@ public Slice slice(int index, int length)
 
     public int indexOfByte(int b)
     {
-        b = b & 0xFF;
+        b &= 0xFF;
```

Comment:
actually a checkArgument would be more appropriate.

Reply:
Agreed, but that's a potentially breaking change, so I'll do that in a separate PR.